precommit/mirsg-hooks.yaml

```diff
@@ -10,11 +10,41 @@ repos:
             --config-data={
               extends: default,
               rules: {
+                anchors: enable,
+                braces: {
+                  forbid: true
+                },
+                brackets: {
+                  forbid: true
+                },
+                colons: enable,
+                commas: enable,
                 comments: {
                   min-spaces-from-content: 1
                 },
+                comments-indentation: enable,
+                document-end: disable,
+                document-start: enable,
+                empty-lines: enable,
+                empty-values: disable,
+                float-values: enable,
+                hyphens: enable,
+                indentation: enable,
+                key-duplicates: enable,
+                key-ordering: disable,
                 line-length: {
-                  level: warning
+                  max: 160
```

Comment:
As @drmatthews pointed out, this is what `ansible-lint` uses https://github.com/UCL-MIRSG/.github/pull/95#issuecomment-1948470053